.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo ":docker::package: Push EOS ubuntu 18.04 base Image to Dockerhub"
+
+# buildkite-agent artifact download someversion.deb .
+
+echo ":download: Downloading artifact from Buildkite step Ubuntu 18.04"
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successfull"
+
+PREFIX='base-ubuntu-18.04'
+SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
+SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
+echo "$SANITIZED_BRANCH"
+echo "$SANITIZED_TAG"
+
+# do docker build
+echo ":docker::build: Building image..."
+
+set -e
```

Comment:
`set -e` is already enabled on line 3. We can remove this.

Reply:
done too.